aten/src/ATen/native/quantized/library.cpp

```diff
@@ -105,11 +105,11 @@ TORCH_LIBRARY(quantized, m) {
   m.def("elu(Tensor self, float output_scale, int output_zero_point, Scalar alpha=1, Scalar scale=1, Scalar input_scale=1) -> Tensor");
   m.def("embedding_bag_prepack(Tensor weight) -> __torch__.torch.classes.quantized.EmbeddingPackedParamsBase W_prepack");
   m.def("embedding_bag_unpack(__torch__.torch.classes.quantized.EmbeddingPackedParamsBase W_prepack) -> Tensor W_origin");
-  m.def("embedding_bag_byte_prepack(Tensor weight) -> Tensor");
+  m.def("embedding_bag_byte_prepack(Tensor weight, bool optimized_qparams=False) -> Tensor");
```

Comment:
When we enable this at the python level, how does the user control which observer is used for embeddings? i.e should we change the default for this op to have optimized_qparams=True?

Reply:
We can have a separate observer that calls the aten op to calculate optimized qparams. 
I'm not sure if setting this to true will have any implications on PyPer perf.